docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.0/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="new-features-7-1-0_{context}"]
+= New features
+
+This section provides the new features and improvements of the {ProductFullName} 7.1.0.
+
+.Support for analyzing applications managed with Gradle has been added to {ProductShortName} 
+
+Previously, you could use the {ProductFullName} to analyze Java applications managed only with Maven. With this update, {ProductShortName} can also extract dependencies from Gradle projects. As a result, you can now analyze applications that use Gradle instead of Maven. 
+
+:FeatureName: Support for analyzing applications managed with Gradle
+
+include::../snippets/developer-preview-admonition.adoc[]
+
+
+
+.Support for analyzing .NET applications has been added to {ProductShortName}
+
+In {ProductFullName} 7.1.0, you can use MTA to analyze Windows-only .NET framework applications to aid migration from version 4.5 or later to multi-platform .NET 8.0 running on OpenShift Container platform. This feature is only available in the command-line interface (CLI).
```

Comment:
In {ProductFullName} 7.1.0, you can use {**ProductShortName**} to analyze Windows-only .NET framework applications to aid migration from version 4.5 or later to multi-platform .NET 8.0 running on {ocp-first} This feature is only available in the command-line interface (CLI).

Plz reconfirm {ocp-first}

Reply:
Good catch, but I'd use {ProductFullName} instead. 

Also, what do you mean by "reconfirm" {ocp-first}?